customer/client_test.go

```diff
@@ -159,18 +159,22 @@ func TestCustomerGet(t *testing.T) {
 func TestCustomerDel(t *testing.T) {
 	res, _ := New(nil)
 
-	err := Del(res.ID)
+	c, err := Del(res.ID)
```

Comment:
Totally fair. I did the customer ones first and later used the xxxDeleted convention. updating now